### PR TITLE
feat(core/pipelines): Add execution breadcrumbs to the pipeline execution component

### PR DESCRIPTION
**Pull request by RodEsp**
This PR adds breadcrumbs that show the parent executions that triggered a given pipeline execution as discussed and shown in [this Slack thread](https://spinnakerteam.slack.com/archives/CH3FMKA3U/p1595534707109400).

<details>
<summary>Screenshots</summary>

image

image
</details>

### CODE REVIEW

**Comment by RodEsp** [comment]
Still working on making the links in the breadcrumbs take you to the stage that actually kicked off the execution in question. Should have that ready relatively soon, at which point I will mark this ready for review.

**Comment by RodEsp** [comment]
Heya @christopherthielen, sorry I'm just getting back to this now. I'm not actively assigned to Spinnaker work at SF anymore but my coworker, [@emagana](https://github.com/emagana) & co, will be "taking over" these efforts.

**Comment by christopherthielen** [comment]
@RodEsp I see you marked this as ready for review, can you confirm that you believe this is all buttoned up and ready to go?

**Comment by RodEsp** [comment]
@christopherthielen I changed it into a normal PR because I wasn't sure if my coworkers would be able to do that themselves or not. The links still don't take you to the specific stage, just the pipeline though.

**Comment by emagana** [comment]
@christopherthielen  Can we try to merge this as it is and file an enhancement after that? As Rodrigo mentioned, he is leading other work at SF. It is better for us to get this merge now and follow up with new PRs. Thank you in advanced!

**Comment by christopherthielen** [comment]
> Can we try to merge this as it is 

Yeah, I'll review this tomorrow and hopefully we can merge it

File: app/scripts/modules/core/src/pipeline/executions/execution/ExecutionBreadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          <ParentExecutionLink execution={execution} />
+          {index !== array.length - 1 && <i className="fas fa-angle-right execution-breadcrumb-marker"></i>}
+        </React.Fragment>
+      ))}
+    </div>
+  );
+};
+
+function ParentExecutionLink({ execution }: IExecutionBreadcrumbsProps) {
+  const { $state } = ReactInjector;
```
Comment (caseyhebebrand) [inline app/scripts/modules/core/src/pipeline/executions/execution/ExecutionBreadcrumbs.tsx:37]: Suggestion: Use uirouter's `useCurrentStateAndParams` hook.

**Review by caseyhebebrand — APPROVED** [review]

**Comment by emagana** [comment]
Awesome! Thanks a lot...

**Pull request merged**